Lesson 1/solutions/3_5.py

```diff
@@ -0,0 +1,14 @@
+# Task 3_5 - sum and product
+
+
+def sum_and_product(list_):
```

Comment:
Какой смысл в подчеркивании в конце аргументов и переменных внутри функций?

Reply:
Для того, чтобы имена переменных не перекрывали ключевые слова. Из PEP 8:
single_trailing_underscore_: used by convention to avoid conflicts with Python keyword, e.g.